server/SocketManager/socketmanager.js

```diff
@@ -1,11 +1,14 @@
 let sockets = {};
+let socket_user_map = {};
 
-const updateSocket = (usertoken, clientsocket) => {
+const updateSocket = (usertoken, clientsocket, user_id) => {
   sockets[usertoken] = clientsocket;
+  socket_user_map[usertoken] = user_id;
 };
 
 const removeSocket = usertoken => {
   delete sockets[usertoken];
+  delete socket_user_map[usertoken];
 };
```

Comment:
The `clientsocket` parameter in the `updateSocket` function is not used. If it's not needed, consider removing it to avoid confusion.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*